### DISCUSSION TITLE
Wie benutzte ich eine ältere FW als basis für Freetz?

### DISCUSSION THREAD

**Discussion by Conan179** [Freetz-NG/freetz-ng#879] [Allgemein] [answered]
Ich möchte für meine 6509 ein freetz bauen, mit einer ältere FW genau genommen diese FRITZ.Box_6590_Cable-07.20.image ich hab sie als image file vorliegen.
Gibt es eine funktion im make menuconfig wo man festlegen kann, das ein bestimmtes image file nutz werden muss?
Downlaoden geht nicht mehr, das Image liegt leider nicht mehr auf dem ftp server.

**Comment by fda77** [comment, marked as answer]
Du wählst da die 7.2x aus:

![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/4794143e-0cad-4f77-a71b-7c0cc3b53bb3)

Und wenn es die bei AVM nicht mehr gibt frag im IPPF oder DEB nach der Datei. Die gehört dann nach dl/fw/

Falls die die neueste Version dieser Reihe nicht passt gibt es dies (nicht nicht nicht nicht nicht empfehlenswert!)

![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/0e958c65-c1ad-4855-bbc5-5ec97bbb6d29)

**Reply by Conan179** [reply]
Das mit dem  "7.2x" geht nicht, avm hat alle älteren FWs nicht mehr auf dem ftp (Hab ich oben schon geschrieben ;) )
[fw_avm_Ftp.txt](https://github.com/Freetz-NG/freetz-ng/files/13209067/fw_avm_Ftp.txt)
Danke für den hinweis, wo man nachfragen kann, in diesem fall, habe ich das image file noch.

**Reply by fda77** [reply]
Wenn du sie noch hast, wie gesagt

> Die gehört dann nach dl/fw/

**Reply by Conan179** [reply]
Habe ich gemacht, das image wird ohne probleme gebaut.
Aber selbst in der minimal konfig (frisch ausgecheckt, keine ändernungen nur fw 7.20) ist es nicht lauffähig, die Fritzbox kann das WLAN nicht starten.

**Reply by fda77** [reply]
Ich versteh den Zusammenhang und Gedankensprung nicht so ganz. Wenn irgendwas nicht geht muss man schauen weshalb es so ist

**Reply by Conan179** [reply]
Es gibt keinen, die AVM konfig war einfach defekt, Werkseinstellungen geladen, geht wieder alles.

**Reply by fda77** [reply]
Wunderbar. Theoretisch kann man bei den Docsis Geräten auf den Arm ein anderes Fos wie auf dem Puma flashen. Praktisch können die dann nicht kommunizieren

**Reply by Conan179** [reply]
Mir geht es darum, mit der 7.20fw, syncronisiert sich die 6590 mit 0/0kbs mit der Kabelkopfstelle, surfen geht nicht klar, aber man kann die werte der KAnnäle abgreifen. alle fws nach der alten 7.20 zeigen dieses verhalten leider nicht, ich hatte es schon mal angesprochen, https://github.com/Freetz-NG/freetz-ng/discussions/272 war aber nichts ganzes und nichts halbes, ich habe dann ein älteres freetzimage von mir genommen mit der 7.20. Gestern habe ich halt mal versucht zu fragen, ob es geht, eine alte fw mit aktuellen freetz zubauen und es geht. Das die fritte probleme mit dem Wlan bekommen hat, hatte nichts mit freetz zu tun, da hat sich irgend was im laufe der Jahre, bei der internen avm konfigs zerschossen, werksteinstellungen geladen, wlan geht wieder.

**Reply by fda77** [reply]
Ach so, die 7.5x geht bei Puma6 gar nicht? Da würde ich zuerst versuchen das Problem einzugrenzen, zB andere die dieses Gerät nutzen fragen ob bei denen 7.5 geht. 7.2 hat immerhin diese unbehobene Sicherheitslücke

**Reply by Conan179** [reply]
Nein, ich hänge mal ein screenshot an, links die 6590 mit fw 7.20, rechts die 6591 mit 7.57, beide sind mesh reapter.
Die 6590 zeigt werte der Docsis Leitung an, die 6591 nicht.
![kabel_info_fritten_011123](https://github.com/Freetz-NG/freetz-ng/assets/1906242/95740da6-ec19-4e1f-9c20-7cba4e3e2fe3)

**Reply by fda77** [reply]
Wie soll denn eine "Repeater" die Werte der Verbindung wissen? Die ist doch nur am "Mesh-Master".
Was bei der 6591 steht kann ich nicht so ganz erkennen

**Reply by Conan179** [reply]
Es steckt ein Coaxkabel an der Modem Buchse dran.
Es geht links um die Menüs, beider 6590 ist Kabel-Information bei der 6591 nicht.
![kabel_info_fritten_011123a](https://github.com/Freetz-NG/freetz-ng/assets/1906242/cb130ca9-2f16-4985-90d0-61949e4365a3)
![kabel_info_fritten_011123b](https://github.com/Freetz-NG/freetz-ng/assets/1906242/95b222d9-d44d-47e9-b33f-524a16526e4b)

**Reply by fda77** [reply]
KapierT! Das liegt dann aber an AVM. Wobei es im Repeater Modus (wie in 7.5) keinen Sinn macht den Modem-Teil zu starten. Das dürfte einiges an Strom brauchen und das Gerät auch einiges wärmer machen. Die Werte müsste der Router doch auch anzeigen

**Reply by Conan179** [reply]
Sag ich doch! es ist eine Eigenart der 7.20fw, alle danach können das nicht mehr.
Das modem (6690) kan es auch anzeigen stimmt, aber das script, das die daten ausliest, ist immer nur auf eine FW konzipiert und wen avm, die webseite ändert, werden keine daten mehr gesammelt und der ersteller des Script vergisst mich mal, die neue version zugeben.

**Reply by Conan179** [reply]
DAs sind z.b. Teile der Werte der Leitung, der letzten 2 Jahre.
![kabel_info_fritten_011123d](https://github.com/Freetz-NG/freetz-ng/assets/1906242/747f6135-f9c2-48fe-a63b-a6b583e7ba1e)

**Reply by fda77** [reply]
Sehr schön, ich würde aber nicht das avm-webif parsen sondern direkt die cmd-befehle auf der Box für docsis verwenden. Ich hab irgendwo gesehen dass man damit alles mögliche abfragen kann. Das geht schneller und bei neuen FOS-Versionen ändert sich selten etwas.
Für DSL hab ich via snmp gemacht:
![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/a7ff6bb2-71d7-4a32-bdd7-e98cbdaaf481)
Und das kann man zb von Caci abfragen lassen (Scheiss Telekom Leitung)
![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/27988656-736c-413b-9653-c755cbd79e02)

**Reply by Conan179** [reply]
DAfür müste man zuerst rausfinden, ob es via snmp überhaupt geht, 1.3.6.1.2.1.10.94.1.1.3 liegert bei meiner docsis box das:
01.11.2023 16:36:15 (3 ms) : SNMP V2c
01.11.2023 16:36:15 (4 ms) : Walk 1.3.6.1.2.1.10.94.1.1.3
01.11.2023 16:36:15 (9 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.1.5 = "0.0.0.0" [ASN_IPADDRESS]
01.11.2023 16:36:15 (14 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.1.6 = "0.0.0.0" [ASN_IPADDRESS]
01.11.2023 16:36:15 (18 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.1.8 = "0.0.0.0" [ASN_IPADDRESS]
01.11.2023 16:36:15 (23 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.1.9 = "0.0.0.0" [ASN_IPADDRESS]
01.11.2023 16:36:15 (27 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.2.5 = "0.0.0.0" [ASN_IPADDRESS]
01.11.2023 16:36:15 (32 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.2.6 = "0.0.0.0" [ASN_IPADDRESS]
01.11.2023 16:36:15 (38 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.2.8 = "0.0.0.0" [ASN_IPADDRESS]
01.11.2023 16:36:15 (43 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.2.9 = "0.0.0.0" [ASN_IPADDRESS]
01.11.2023 16:36:15 (47 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.3.5 = "2" [ASN_INTEGER]
01.11.2023 16:36:15 (51 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.3.6 = "3" [ASN_INTEGER]
01.11.2023 16:36:15 (57 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.3.8 = "2" [ASN_INTEGER]
01.11.2023 16:36:15 (62 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.3.9 = "2" [ASN_INTEGER]
01.11.2023 16:36:15 (67 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.4.5 = "0" [ASN_INTEGER]
01.11.2023 16:36:15 (72 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.4.6 = "0" [ASN_INTEGER]
01.11.2023 16:36:15 (77 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.4.8 = "64" [ASN_INTEGER]
01.11.2023 16:36:15 (82 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.4.9 = "0" [ASN_INTEGER]
01.11.2023 16:36:15 (88 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.5.5 = "1" [ASN_INTEGER]
01.11.2023 16:36:15 (93 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.5.6 = "1" [ASN_INTEGER]
01.11.2023 16:36:15 (98 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.5.8 = "1" [ASN_INTEGER]
01.11.2023 16:36:15 (104 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.5.9 = "1" [ASN_INTEGER]
01.11.2023 16:36:15 (109 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.6.5 = "0" [ASN_INTEGER]
01.11.2023 16:36:15 (114 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.6.6 = "0" [ASN_INTEGER]
01.11.2023 16:36:15 (120 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.6.8 = "0" [ASN_INTEGER]
01.11.2023 16:36:15 (125 ms) : 1.3.6.1.2.1.10.94.1.1.3.1.6.9 = "0" [ASN_INTEGER]

**Reply by fda77** [reply]
https://mibs.observium.org/mib/ADSL-LINE-MIB/ das ist halt dsl upstream
Du musst snmp aber noch mit daten füttern, das weiss weder wie man dsl noch docsis ausliest. Dafür wären die docsis befehle von avm gut von denen ich schrieb

**Comment by Prisrak1** [comment]
Schön zu wissen, die Version 6590_07.20.ger_freetz-ng.mage lässt sich bauen.

**Reply by fda77** [reply]
Wenn sich das mal ändert sag bescheid, alte Versionen teste ich gar nicht.

Hast du eigentlich das neue "AVM-DocsisState" bei dem "AVM-Logs" gesehen? Wenn ich Kabel hätte und das loggen möchte würde ich das parsen

**Reply by Conan179** [reply]
"AVM-DocsisState"?

**Reply by fda77** [reply]
Wenn es den Punkt nicht gibt ist es keine Docsis Fritzbox oder Freetz ist zu alt

**Reply by Conan179** [reply]
Wow nice, kann man das auch exportieren? Um es z.b. als txt abzuspeichern?

**Reply by Prisrak1** [reply]
Ich hab's hochgeladen, kann einer mal gerne testen, wenn er´s kann. Sonst wüsste ich irgendwie nicht was ich damit für euch als Ergebnis posten kann. [6590_07.20.ger_freetz..image.zip]

**Reply by fda77** [reply]
@Prisrak1  Danke, ich kann ein Image selbst bauen... da vermutlich auch euer Addon drin ist würde es mir (und jedem anderen) die Config zerhauen
@Conan179 Du kannst auch einfach den Shellbefehl ausführen der dazu benutzt wird

**Reply by Prisrak1** [reply]
Da habe ich keine Zweifel, War eigentlich nur eine Hilfestellung für die wenigen für uns, die es nicht können. :)

**Reply by fda77** [reply]
Mit eurem Oscam Addon bekommt man aber nur Probleme, das neueste dessen Ursache das Addon ist https://www.ip-phone-forum.de/threads/317710/

**Reply by Prisrak1** [reply]
Aus meiner Sicht ist es kein Problem und am Oscam Addon liegt es nicht. Wenn daraus jemand eine Diskussion eben herstellen mag, muss ich mich dazu überhaupt nicht äußern.

**Reply by fda77** [reply]
Welche "Sicht"? "ich definiere mich als Feature und nicht als Fehler"? 
Wenn man in den Quellcode reinschaut sieht man die Ursache in eurem Addon. Das ist Fakt. Euer Addon ist schon seit langem als sehr fehlerbehaftet bekannt

**Reply by fda77** [reply]
Und ganz ehrlich, ich bin den endlosen Schwachsinn mit euerem Addon leid

**Reply by Conan179** [reply]
@fda77 Danke fürs einbauen vom AVM-DocsisState ist ne schicke sache, dazu eine frage, ich lasse sie mir momentan per cron als txt file auf den internen speicher ablegen, kannst du mir helfen, das der inhalt per email verschickt wird? Also den inhalt des files als email text,

**Reply by fda77** [reply]
Man kann das PushMail von AVM nutzen. Ich glaube im IPPF hat zum "mailer" jemand was längeres geschrieben, ich kann es aber nicht finden

```
$ mail -?

Usage: mail {send|sendonce|enqueue} [MAILER_OPTION]...
       mail {process|config|list}

usage: mailer [-s subject] -f from -t to -m mailserver [-a authname [-w passwd]] -i file(s) [-r] [-d attachfile(s)] [-l] [-c charset]
options:
  -?                 - print this help
  -s STRING          - subject. ("FRITZ!Box")
  -f STRING          - from. (NULL)
  -t STRING          - to. (NULL)
  -m STRING          - mailserver. (NULL)
  -a STRING          - authname. (NULL)
  -w STRING          - passwd. (NULL)
  -n                 - no SMTP-Auth. (NOTSET)
  -d STRING          - attachment(s). (NULL)
  -i STRING          - inline part(s) - add text file(s) here. (NULL)
  -r                 - use "multipart/related" if possible. (NOTSET)
  -l                 - SMTP with SSL or TLS. (NOTSET)
  -c STRING          - charset (default: iso-8859-1, no charset conversion!). (NULL)
note: please use filename extensions if possible
```

Das funktioniert bei mir, auch mit weniger Parametern als angeblich nötig. :
```
mailer  -s "Teeest - $(date)"  -d /tmp/eine.datei
```

**Reply by Conan179** [reply]
Ok, das funktioniert, aber ich hab es jetzt erst bemerkt, für meine zwecke gar nicht funktioniert. Der Datei Name ändert sich jedes mal, wen ich "ctlmgr_ctl u showdocsisstate > /var/media/ftp/docsiswerte/logs.docsisstate_`date +%Y%m%d_%H%M`.txt 2>&1" per cron aufrufen lasse.

**Reply by fda77** [reply]
Ruf am besten mit cron ein Script auf, leg es zb in /tmp/flash. Darin machst du eine Variable mit dem Dateinamen. Die Variable benutzt du dann mit shwodocsisstats und dem mailer

**Reply by Conan179** [reply]
Ok, dann muss ich mich erst mal einlesen wie ein script geschrieben wird, hab davon keinen schimmer 😅

**Reply by fda77** [reply]
da kommen einfach die befehle von oben rein:
```
#!/bin/sh
LOG="/var/media/ftp/docsiswerte/logs.docsisstate_$(date +%Y%m%d_%H%M).txt"
ctlmgr_ctl u showdocsisstate > $LOG 2>&1
mailer  -s "Teeest - $(date)"  -d $LOG
```
und das script mit "chmod +x "/var/media/ftp/docsiswerte/script.sh" ausführbar machen

**Reply by Conan179** [reply]
ok ich danke dir für deine Hilfe.

**Reply by Conan179** [reply]
Hmmm geht nicht.
Wen ich versuche das script zu starten, kommt das:
root@fritz:/var/mod/root# /var/media/ftp/docsiswerte/doc_stat.sh
-sh: /var/media/ftp/docsiswerte/doc_stat.sh: Permission denied
root@fritz:/var/mod/root# cd /var/media/ftp/docsiswerte/
root@fritz:/var/media/ftp/docsiswerte# ./doc_stat.sh
-sh: ./doc_stat.sh: Permission denied
root@fritz:/var/media/ftp/docsiswerte#

getestet mit rechte oktal 555 und 777

**Reply by fda77** [reply]
AVM lässt es nicht zu dass von usb Dateien gestartet werden. Entweder 
 - https://freetz-ng.github.io/freetz-ng/patches/ DROP_NOEXEC_EXTERNAL aktivieren
 - oder das Script doch unter /tmp/flash/ (+modsave) speichertn

**Reply by Conan179** [reply]
AVM zählt den internen Speicher zum USB? ok, ich werde ein neues Image für meine boxen machen, danke.

**Reply by fda77** [reply]
Ich glaube alles unter /var/media/ftp ... die Fehlermeldung ist jedenfalls typisch dafür. Du kannst auch mit "mount" schauen wie es gemountet ist ("noexec"), und mit "mount -o remount"  ummounten

**Reply by Conan179** [reply]
@fda77 hab für meine 6690 ein neues image bauen lassen, mit dem Patch, irgend was hat sie nicht vertragen und macht jetzt nen bootloop. gibt es push_image entgeltlich auch als Windows version?

**Reply by Prisrak1** [reply]
Aus meiner Erfahrung liegt es an den folgenden Einstellungen: Developer,  Other patches, Disable AVM watchdog. Bitte testen.

**Reply by Conan179** [reply]
@Prisrak1 ich hab nicht dich gefragt, ich werde nichts mehr lesen was von dir kommt.,

**Reply by Prisrak1** [reply]
Sorry für deine Empfindlichkeit. Wüste gar nicht womit ich es verdient habe, aber danke für dein Feedback

**Reply by fda77** [reply]
avm-watchdog macht genau bei 1 Gerät Probleme, der 5530. Diese ist aber selbst nur als "developer" auswählbar, und wird es wahrscheinlich immer bleiben / wieder rausgeworfen werden.
Für Windows ist mir an öffentlichen Dingen nur nur peterpawns powershell script und das veraltete ru-tool bekannt. PPs script kann glaub ich kein docsis sondern nur "inmemory" flashen. 
Aber es solle reichen das "linux_fs_start" zu ändern um das ältere Image zu booten - das kann man eigentlich auch mit Windows via ftp machen oder (vermutlich) mit PPs script.

**Reply by Conan179** [reply]
So inzwischen ist meine box wieder online, hab das vorgehende image über ein live linux geflasht.
In der /var/flash/crash.log steht:
```
Uptime: 1032106
Log-Version: 2.1
Buildnumber: 107843
Firmware: 07.57
Buildtype: 1
Builddirty: 0
```

```
Signal Segmentation fault in crashdump: show info (2)
2024-01-10 15:50:47(2) [Segmentation fault] avmnexusd([1854/32052]) CRASHED at 0+0x677816db (/lib/libavmcsock.so.2 at 0001c6db) accessing 00000006
SIGNO 11 ERRNO 0 CODE 1
eax: 00000002  ebx: 67805148  ecx: 66fb4d0b
edx: 6778107f  esi: 00000001  edi: 00000080
ebp: 66fb4c78  esp: 66fb4bd0  eip: 677816db
cs :     0073  ds :     007b  es :     007b
fs :     0000  gs :     0033  ss :     007b
err: 00000004  oms: 00000000  tno: 0000000e (segfault is fixable)
esp: 66fb4bd0  cr2: 00000006
eflags [00010202] PF IF RF 
FA 00000006
```

In der /var/flash/panic steht
[panic .txt](https://github.com/Freetz-NG/freetz-ng/files/13955087/panic.txt)

Ich werde es vorerst mal lassen, ich hab es eben nur mit sehr viel mühe und tricksen hinbekommen, den nexten versuch mache ich erst, wen ich ein linux system parat habe.

Zum thema "linux_fs_start", das hat bei mir beim flashen einer kabel box (6590,6591 und 6690) nie funktioniert.
![freetz-ng_160124](https://github.com/Freetz-NG/freetz-ng/assets/1906242/8418251d-93f9-4689-bae2-4e41a6185c4f)

**Reply by Prisrak1** [reply]
.. von fda77: .. PPs script kann glaub ich kein docsis sondern nur "inmemory" flashen. ... "

.. 

Bei Bedarf des aktiven Partitionsset auslesen (Variable "linux_fs_start") per folgendem Kommando:
```
c:\YourFritz\master\eva_tools\EVA-FTP-Client.ps1 -ScriptBlock { GetEnvironmentValue linux_fs_start }
```

Den Wert der Variable linux_fs_start (hier im Beispiel "1") bei Bedarf ändern wenn man die Firmware im anderen (derzeit inaktiven) Partitionsset installieren möchte (ansonsten diesen Schritt einfach überspringen):

```
c:\YourFritz\master\eva_tools\EVA-FTP-Client.ps1 -Verbose -ScriptBlock { SwitchSystem }
```

[6690 6660 6591 flashen:]
```
.\EVA-FTP-Client.ps1 -ScriptBlock { UploadFlashFile .\part_03_ATOM_ROOTFS.bin mtd0 }
.\EVA-FTP-Client.ps1 -ScriptBlock { UploadFlashFile .\part_02_ATOM_KERNEL.bin mtd1 }
.\EVA-FTP-Client.ps1 -ScriptBlock { UploadFlashFile .\part_09_ARM_ROOTFS.bin mtd6 }
.\EVA-FTP-Client.ps1 -ScriptBlock { UploadFlashFile .\part_08_ARM_KERNEL.bin mtd7 }
.\EVA-FTP-Client.ps1 -ScriptBlock { SetEnvironmentValue linux_fs_start 0 }
```

oder

```
.\EVA-FTP-Client.ps1 -ScriptBlock { UploadFlashFile .\part_03_ATOM_ROOTFS.bin 'mtd;' }
.\EVA-FTP-Client.ps1 -ScriptBlock { UploadFlashFile .\part_02_ATOM_KERNEL.bin 'mtd<' }
.\EVA-FTP-Client.ps1 -ScriptBlock { UploadFlashFile .\part_09_ARM_ROOTFS.bin 'mtd=' }
.\EVA-FTP-Client.ps1 -ScriptBlock { UploadFlashFile .\part_08_ARM_KERNEL.bin 'mtd>' }
.\EVA-FTP-Client.ps1 -ScriptBlock { SetEnvironmentValue linux_fs_start 1 }
```

```
.\EVA-FTP-Client.ps1 -ScriptBlock { RebootTheDevice }
```
- PowerShell Fenster schließen .. "

**Reply by fda77** [reply]
@Prisrak1: Danke für die Befehle. Diese gelten aber nur für "uimg", also die neueren Docsis wie 6690 6660 6591. Die alten "dual-boot" Docsis 6490 6590  haben andere mtd Namen: https://github.com/Freetz-NG/freetz-ng/blob/master/tools/push_firmware#L127 Um genau zu sein sind im 1. Partitions set "0" ARM und Atom vertauscht! Das sind dann evlt die "image kaputt" / "bootet nicht" usw Fehler

**Reply by Prisrak1** [reply]
Ja, das ist uns beiden eben seit langem bekannt.

**Reply by fda77** [reply]
@Conan179 "UNKNOWN" heisst dass diese Partition nicht gemountet werden konnte. Entweder ist der Inhalt "defekt" zB nicht komplett übertragen/geschrieben wurden oder in einem Format das nicht erwartet wurde - zb Atom/Arm vertauscht...
In deinen Logs sieht man "avmnexusd" und einen "BUG: unable to handle kernel NULL pointer dereference at   (null)" direkt beim Booten, keine Ahnung woher das kommt. Bei den Docsis könnte es auch vorkommen wenn man schlecht flash, dass ARM und Atom nicht zueinandere passen zB verschieden FOS versionen, dann crasht auch irgend was

**Reply by Conan179** [reply]
Als ich die 6690 zum ersten mal gefreetz hatte, (per push_firmware) musste ich die bootbank manuell umstellen, beim flash vorgang hatte es nicht funktioniert, da bin ich mir 100% sicher, das selbe verhalten hatte ich bei der 6591 und vor vielen monaten auch auf der 6590 (https://github.com/Freetz-NG/freetz-ng/issues/60) flash vorgang ohne probleme, umschaltung hat nicht so gewollt.

Ich hab gestern für 4 Boxen von mir ein neues image erstellt, mit dem DROP_NOEXEC_EXTERNAL patch, beim make für die 7272/6590/6591 ging es sehr schnell, beim image für die 6690 seltsamerweise nicht, als hätte der make vorgang das image kommplet neu erstellen müssen (wie bei einer neuen FW).
Alle image per freetz interface geflasht. Merkwürdig oder?

**Reply by fda77** [reply]
Ne, es sind ja verschieden Geräte... Du musst schauen was genau compiliert wird und was sich seit letztem mal geändert hat

**Reply by Conan179** [reply]
Aha und wie mache ich das? Beim Build vorgang flitzten sehr viele infos sehr schnell vorbei...

**Reply by fda77** [reply]
Ich scrolle einfach nach oben... im IPPF wird meist [fmake](https://github.com/Freetz-NG/freetz-ng/blob/master/fmake) empfohlen, hab ich aber nie verwendet

**Comment by superemil70** [comment]
> Mit eurem Oscam Addon bekommt man aber nur Probleme, das neueste dessen Ursache das Addon ist https://www.ip-phone-forum.de/threads/317710/

Ist schon geil, das bei Probleme in Freetz das oscam addon Schuld ist. Auch wenn es nicht verbaut ist

@FDA,

du bist und bleibst ein Spinner. Mehr fällt mir zu dir nicht ein

**Reply by fda77** [reply]
https://freetz-ng.github.io/freetz-ng/ADDONS.html#nervt-mit-dem-krempel-nicht-herum

**Comment by Prisrak1** [comment]
Es betrübt mich besonders, wenn jemand in der Kommunikation persönlich werdend und unangebrachte Worte benutzt, ungeachtet dessen, ob sie der Realität entsprechen oder nicht. Ich würde darum bitten, sich auf einer Ebene des gegenseitigen Respekts zu bewegen, auf der eine respektvolle Kommunikation möglich ist. Ein vorbildhaftes Verhalten in diesem Sinne wäre sehr wünschenswert.
